### DISCUSSION TITLE
No real stereo audio in server example

### DISCUSSION THREAD

**Discussion by exzhawk** [aiortc/aiortc#604] [Q&A]
Step to reproduce:
get the example in `examples/server`
replace `demo-instruct.wav` with a stereo audio file. attached file is a stereo audio file with muted right channel.
[demo-instruct.zip](https://github.com/aiortc/aiortc/files/7174660/demo-instruct.zip)
run the demo normally.
although the right channel in the file is muted, there is sound in right channel.

**Comment by FabienDanieau** [comment]
Hi, I have a similar issue with the webcam example. If I use a stereo file as input (for instance https://www.axel-hahn.de/axel/download/lets-play_2.0_.ogg), the webpage plays a mono sound. The two channels are mixed together.
```
python webcam.py --play-from ~/Music/lets-play_2.0_.ogg
```
Any idea how to solve this? Thx

**Reply by FabienDanieau** [reply]
Got it. By default Opus is mono although there are two channels. The stereo option in the SDP should be explicitly set:
https://stackoverflow.com/a/58898298
